scripts/build-setup.sh

```diff
@@ -244,10 +242,8 @@ if run_step "6"; then
         pushd $CYDIR/sims/firesim &&
         (
             set -e # Subshells un-set "set -e" so it must be re enabled
-            echo $CYDIR
             source sourceme-manager.sh --skip-ssh-setup
             pushd sim
-            make target-classpath
```

Comment:
Removed since this is a power-user target. Technically, the target classpath in FireSim should only be built for FireSim projects (for example: `midasexamples`)

Reply:
Is this a power-user target? I thought this was done to speed up the first firesim build.

Reply:
Edit: This is *now a power user target. FireSim should now call out to the Chipyard makefile (which should already be precompiled).